ckanext/notify/controllers/ui_controller.py

```diff
@@ -259,6 +288,15 @@ def send_slack_notification(self, template, result):
                             'datarequest_url': result['datarequest_url'],
                             'datarequest_title': result['title'],
                             'datarequest_description': result['description'],
+                            'action_type': template,
+                        }
+
+            email_subject = base.render_jinja2('notify/email/{}.txt'.format('subject'), extra_vars)
+            email_body = base.render_jinja2('notify/email/{}.txt'.format(template), extra_vars)
+
+            for channel in channels:
+                channel = dotdict(channel)              
+                mailer.mail_user(channel, email_subject, email_body)
```

Comment:
How does this work in the `send_slack_notification` method? I'd figure email were separate from Slack.